src/doc_scan_service/doc.scan.constants.js

```diff
@@ -13,6 +13,9 @@ module.exports = Object.freeze({
   ID_DOCUMENT_FACE_MATCH: 'ID_DOCUMENT_FACE_MATCH',
   ID_DOCUMENT_COMPARISON: 'ID_DOCUMENT_COMPARISON',
   THIRD_PARTY_IDENTITY: 'THIRD_PARTY_IDENTITY',
+  WATCHLIST_SCREENING: 'WATCHLIST_SCREENING',
+  ADVERSE_MEDIA: 'ADVERSE-MEDIA',
```

Comment:
How come this is the only Dc scan constant value with a `-` instead of `_`.  I expect this was something done on the Doc Scan API side so we can't avoid it here.

Reply:
Yup, good spot! I checked again the API, and indeed it is expecting `'ADVERSE-MEDIA'` which seems to be a break in consistency.

Reply:
@taranchauhan @laurent-yoti This constant actually comes from the Comply Advantage API, and not directly from our own (hence why it breaks convention).